tests/page-objects/default/contacts/contacts.wdio.page.js

```diff
@@ -56,6 +56,7 @@ const formTitle = () => $('#form-title');
 const contactCardTitle = () => $('.inbox .content-pane .material .body .action-header');
 const contactInfoName = () => $('.content-pane .material .body .card .row .heading-content');
 const contactMedicID = () => $('#contact_summary .cell.patient_id > div > p');
+const contactStatus = () => $('.content-pane .material .body .card .row .heading-content > div');
```

Comment:
This looks fragile and very similar to `contactInfoName`, do you think we can add a `test-id` like we did before? 
I can help if you need :)

Reply:
Let me try it, I want to try without any help. 
I will let you know, thank Jenny!